internal/spdxlicense/generate/generate_license_list.go

```diff
@@ -31,6 +31,15 @@ var licenseIDs = map[string]string{
 	{{ printf "%q" $k }}: {{ printf "%q" $v }},
 {{- end }}
 }
+
+// urlToLicense maps license URLs from the seeAlso field to license IDs and names
+var urlToLicense = map[string]struct {
```

Comment:
Since this is just ID now, I think this could be `map[string]string`, which would simplify a little more stuff throughout this PR

Reply:
Sending that up now